acceptance/config/beaker/options.rb

```diff
@@ -8,6 +8,7 @@
   "puppetservice" => 'puppetserver',
   "use-service" => true,
   "master-start-curl-retries" => 60,
-  "puppetpath" => "/etc/puppetlabs/agent"
+  "puppetpath" => "/etc/puppetlabs/puppet",
+  "puppetserver-confdir" => '/etc/puppetlabs/puppetserver/conf.d'
```

Comment:
Is there a plan for getting this value into Beaker itself? I see this is already the value for PE: https://github.com/puppetlabs/beaker/blob/master/lib/beaker/host/unix.rb#L22 (the foss/aio defaults are different though)